scripts/fetch-data

```diff
@@ -100,6 +108,12 @@ function fetchMunicipalBoundariesData() {
     || : # Don't exit script if non-zero exit code
 }
 
+function fetchUtilities() {
+    aws --profile=iow-boundary-tool s3 cp $IOW_INITIALIZATION_BUCKET/$UTILITIES_SQL $DJANGO_DATA_DIR/
+    docker-compose run --entrypoint "/bin/bash -c" --rm django \
+        "psql --host=database --username=iow-boundary-tool --dbname=iow-boundary-tool --single-transaction --file=$DATA_DIR/$UTILITIES_SQL"
```

Comment:
Is this script only used in development? If so, considering specifying the password here so that the user is not prompted for it when this is run.

Reply:
3a9a3fb